src/polymer-project.ts

```diff
@@ -244,6 +222,69 @@ class HtmlSplitter extends Transform {
       callback(null, file);
     }
   }
+
+  _splitScript(file: File, doc: parse5.ASTNode, filePath: string) {
+    const scriptTags = dom5.queryAll(doc, HtmlSplitter.isInlineScript);
+    for (let i = 0; i < scriptTags.length; i++) {
```

Comment:
can you clean up any differences between `splitScript()` & `splitStyle()`? For example, `splitScript` is using a `for()` loop while `splitStyle()` is using `styleTags.forEach()`.